### PR TITLE
Remove viewportSize in multi-icon-layer's shader and fix getText

### PR DESCRIPTION
**Pull request by jianhuang01**

### CODE REVIEW

File: src/experimental-layers/src/text-layer/text-layer.js [hunk 1]
```diff
@@ -63,7 +63,7 @@ export default class TextLayer extends CompositeLayer {
   }
 
   updateState({props, oldProps, changeFlags}) {
-    if (changeFlags.dataChanged || changeFlags.updateTriggers.getText) {
+    if (changeFlags.dataChanged || props.getText) {
```
Comment (jianhuang01) [inline src/experimental-layers/src/text-layer/text-layer.js]: @ibgreen not sure if it is the right fix, cause I don't know the intention of this check.

Comment (ibgreen) [reply src/experimental-layers/src/text-layer/text-layer.js]: It should be `changeFlags.updateTriggersChanged.getText`.

Good catch!

Comment (georgios-uber) [inline src/experimental-layers/src/text-layer/text-layer.js]: Don't change this line.

Comment (jianhuang01) [reply src/experimental-layers/src/text-layer/text-layer.js]: @georgios-uber it breaks without changing, ib added a suggestion. Is that OK?

Comment (georgios-uber) [reply src/experimental-layers/src/text-layer/text-layer.js]: OK

**Comment by 1chandu** [comment]
@jianhuang01  Given we are publishing `experimental-layers` from master, we don't have to port this to 5.0-release branch.

**Pull request merged**